src/static/netsuke/assets/css/himotoshi.css

```
@@ -274,6 +288,254 @@ a.sidebar-link--sub.active {
+  font-size: 0.75rem;
+  font-weight: 700;
+  letter-spacing: 0.08em;
+  margin-bottom: 1rem;
+  text-transform: uppercase;
+}
+
+.hm-config-keys__layout {
+  /* A subgrid overrides the parent's gutters with its own, so both
+     columns must name the same row gap or their rows drift apart. */
+  --hm-config-keys-row-gap: 0.75rem;
+  display: grid;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)